h/static/scripts/annotator/guest.coffee

```diff
@@ -35,21 +48,10 @@ module.exports = class Guest extends Annotator
       </div>
     '''
 
-  constructor: (element, options, config = {}) ->
-    options.noScan = true
+  constructor: (element, options) ->
     super
-    delete @options.noScan
 
-    # Are going to be able to use the PDF plugin here?
-    if window.PDFTextMapper?.applicable()
-      # If we can, let's load the PDF plugin.
-      @options.PDF = {}
```

Comment:
So what now actually causes the PDF plugin to be loaded on PDF documents? I can't atm see how/where this happens.